File: SDKs/googleapis.com/displayvideo/v1/python/src/sdk/models/shared/inventorysourcegroupassignedtargetingoptiondetails.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class InventorySourceGroupAssignedTargetingOptionDetails:
    r"""InventorySourceGroupAssignedTargetingOptionDetails
    Targeting details for inventory source group. This will be populated in the details field of an AssignedTargetingOption when targeting_type is `TARGETING_TYPE_INVENTORY_SOURCE_GROUP`.
    """
    
    inventory_source_group_id: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('inventorySourceGroupId') }})
    

File: SDKs/googleapis.com/displayvideo/v1/typescript/src/sdk/models/shared/inventorysourcegroupassignedtargetingoptiondetails.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// InventorySourceGroupAssignedTargetingOptionDetails
/** 
 * Targeting details for inventory source group. This will be populated in the details field of an AssignedTargetingOption when targeting_type is `TARGETING_TYPE_INVENTORY_SOURCE_GROUP`.
**/
export class InventorySourceGroupAssignedTargetingOptionDetails extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=inventorySourceGroupId" })
  inventorySourceGroupId?: string;
}
